scripts/system/create/modules/brokenUrlReport.js

```diff
@@ -0,0 +1,390 @@
+    brokenUrlReportContent = brokenUrlReportContent + "                <td class='header'>Name &amp; ID</td>\n";
+    brokenUrlReportContent = brokenUrlReportContent + "                <td class='header'>&nbsp;</td>\n";    
+    brokenUrlReportContent = brokenUrlReportContent + "                <td class='header'>Property</td>\n";
+    brokenUrlReportContent = brokenUrlReportContent + "                <td class='header'>Status</td>\n";
+    brokenUrlReportContent = brokenUrlReportContent + "                <td class='header'>Current URL</td>\n";
+    brokenUrlReportContent = brokenUrlReportContent + "            </tr>\n";
+    for (var i = 0; i < brokenUrlReportInvalideUrlList.length; i++ ){
+        brokenUrlReportContent = brokenUrlReportContent + "            <tr>\n";
+        brokenUrlReportContent = brokenUrlReportContent + "                <td style='color: #999999; width: 5%;'>" + (i + 1) + "</td>\n";
+        brokenUrlReportContent = brokenUrlReportContent + "                <td style='width: 10%;'>" + brokenUrlReportInvalideUrlList[i].type + "</td>\n";
+        brokenUrlReportContent = brokenUrlReportContent + "                <td style='width: 40%;'>" + brokenUrlReportInvalideUrlList[i].name + "<br><font color='#999999'>" + brokenUrlReportInvalideUrlList[i].id + "</font></td>\n";
+        brokenUrlReportContent = brokenUrlReportContent + "                <td style='width: 2%;'><a href='' onclick='selectEntity(" + '"' + brokenUrlReportInvalideUrlList[i].id + '"' + "); return false;'>&#9998;</a></td>\n";        
+        brokenUrlReportContent = brokenUrlReportContent + "                <td style='color: " + brokenUrlReportGetUrlTypeColor(brokenUrlReportInvalideUrlList[i].urlType) + "; width: 10%;'>" + brokenUrlReportInvalideUrlList[i].urlType + "</td>\n";
+        brokenUrlReportContent = brokenUrlReportContent + "                <td style='background-color: #FF0000; color: #FFFFFF; width: 8%;'>" + brokenUrlReportInvalideUrlList[i].validity + "</td>\n";
+        brokenUrlReportContent = brokenUrlReportContent + "                <td style='word-wrap: break-word; width:200px;'>" + brokenUrlReportInvalideUrlList[i].url + "</td>\n";
+        brokenUrlReportContent = brokenUrlReportContent + "            </tr>\n";
+    }
+    brokenUrlReportContent = brokenUrlReportContent + "        </table>\n";
+    brokenUrlReportContent = brokenUrlReportContent + "<div style='width: 100%; text-align: left;'><br>" + brokenUrlReportUrlList.length + " URL tested.<br><br>";
+    brokenUrlReportContent = brokenUrlReportContent + "<hr><font style='font-size: 11px; font-weight: 500;'><i>This report ignores URL from the Asset Server (atp://), local drive paths or any string not starting by 'http'.</i></font></div>";
+    
+    Script.setTimeout(function () {
+        brokenUrlReportOverlayWebWindow.emitScriptEvent(brokenUrlReportContent);
+    }, 3000);
+}
+
+function brokenUrlReportGetUrlTypeColor(urlType) {
+    var hexaColor = "#FFFFFF";
```

Comment:
```suggestion
    var hexColor = "#FFFFFF";
```
Or: `color = "#FFFFFF";`